service/src/main/java/bio/terra/workspace/app/controller/ReferencedGcpResourceController.java

```diff
@@ -189,8 +196,15 @@ public ResponseEntity<Void> deleteGcsObjectReference(UUID workspaceUuid, UUID re
     AuthenticatedUserRequest userRequest = getAuthenticatedInfo();
     workspaceService.validateWorkspaceAndAction(
         userRequest, workspaceUuid, SamWorkspaceAction.DELETE_REFERENCE);
+    var userStatusInfo =
+        SamRethrow.onInterrupted(
+            () -> samService.getUserStatusInfo(userRequest), "get user status info from SAM");
```

Comment:
Could you DRY this out by passing the userRequest into the method and have it do the Sam call?
Like you do on the create call.

Reply:
ah, i passed in the user request in the create call because i want to pass it in to the flight and we make the sam call in the hook in that case...

since Zach just moved all the sam checkAuz call to controller, I wasn't sure if I should re-introduce sam call back to the service.  but sure i can do that